crates/nu-command/tests/commands/select.rs

```diff
@@ -159,3 +159,15 @@ fn selects_many_rows() {
         assert_eq!(actual.out, "2");
     });
 }
+
+#[test]
+fn select_ignores_errors_succesfully() {
+    let actual = nu!(
+        cwd: ".", pipeline(
+        r#"
+        [{a: 1, b: 2} {a: 3, b: 5} {a: 3}] | select -i b; [{a: 1} {a: 2} {a: 3}] | select -i b
```

Comment:
I think it would be better to have them as separate tests. I just wanted to have a test where you `select` a column that doesn't exist at all (in the first example, the `b` column is still present in some rows).